Modules/Loadable/Transforms/Resources/UI/qSlicerTransformsModuleWidget.ui

```diff
@@ -170,6 +176,57 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="ctkCollapsibleGroupBox" name="CenterOfTransformationGroupBox">
+        <property name="enabled">
+         <bool>false</bool>
+        </property>
+        <property name="title">
+         <string>Center of transformation</string>
+        </property>
+        <layout class="QGridLayout" name="gridLayout">
+         <item row="0" column="1">
+          <widget class="qMRMLSpinBox" name="CenterOfTransformationPASpinBox">
```

Comment:
We should store the coordinates in local (transform parent) coordinate system but offer the option to show the coordinate values in either local or world coordinate system (same way as it is done for markup control point positions).

Reply:
Added a new combobox.